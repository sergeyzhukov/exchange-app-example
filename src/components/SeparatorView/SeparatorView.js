import React from 'react'
import { StyleSheet, View } from 'react-native'

export default () => (
  <View style={styles.separator} />
)

const styles = StyleSheet.create({
  separator: {
    backgroundColor: '#C8C7CC',
    height: StyleSheet.hairlineWidth,
    marginLeft: 16,
  },
})
