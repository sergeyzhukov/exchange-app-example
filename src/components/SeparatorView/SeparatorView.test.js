import React from 'react'
import renderer from 'react-test-renderer'
import SeparatorView from './SeparatorView'

test('renders correctly', () => {
  const component = <SeparatorView onPress={() => {}} />
  const tree = renderer.create(component).toJSON()
  expect(tree).toMatchSnapshot()
})
