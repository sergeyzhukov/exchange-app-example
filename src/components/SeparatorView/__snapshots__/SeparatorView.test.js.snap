// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#C8C7CC",
      "height": 0.5,
      "marginLeft": 16,
    }
  }
/>
`;
